content/docs/tutorials/get-started/data-versioning.md

```diff
@@ -251,10 +164,86 @@ $ dvc pull
 > [Sharing Data and Model Files](/doc/use-cases/sharing-data-and-model-files)
 > for more on basic collaboration workflows.
 
-## Other ways to track data
+## Making changes
+
+When you make a change to a file or directory, run `dvc add` again to track
+changes:
+
+```dvc
+$ dvc add data/data.xml
+```
+
+<details>
+
+### ⚙️ Expand to make some changes
```

Comment:
Should this be placed before the dvc add cmd above?

Reply:
yep! I'll make the change

Reply:
done!

Reply:
Thanks!